internal/tools/aws/route53.go

```diff
@@ -78,6 +78,11 @@ func (a *Client) DeleteCNAME(dnsName string, logger log.FieldLogger) error {
 		return err
 	}
 
+	if len(dnsName) >= 64 {
+		logger.Warnf("DNS name too long, skipping since it will never created record-name=%s", dnsName)
```

Comment:
Minor nitpick on the error wording, "skipping since it could never have been created"

Reply:
done thanks!